guides/cookbook.md

```diff
@@ -82,6 +82,94 @@ implied) implementation.
 ## Reading and Writing Ion Data
 
 <script>writeTabs()</script>
+<div class="tabpane C" markdown="1">
+The following example shows how text Ion data can be read from a string:
+```c
+#include <cstdlib>
```

Comment:
I thought cstdlib was a C++ library and not a C library. The point might be mood here but we might want to be explicit on what compiler and libs we are assuming for the examples.